### PR TITLE
community/afl: enable armhf

### PR DESCRIPTION
**Pull request by J0WI**
Fails on x86

### CODE REVIEW

**Comment by TBK** [comment]
Any idea why x86 is failing? Is there a bug issue tracking this?

https://build.alpinelinux.org/buildlogs/build-3-10-x86/community/afl/afl-2.52b-r0.log

**Comment by J0WI** [comment]
AFAIK has the project no bug tracker.

**Comment by TBK** [comment]
> To send bug reports, feature requests, or chocolate, simply drop a mail to lcamtuf@coredump.cx.

File: community/afl/APKBUILD [hunk 1]
```diff
@@ -2,26 +2,24 @@
...
 pkgname=afl
 pkgver=2.52b
-pkgrel=0
+pkgrel=1
 pkgdesc="Fuzzer relying on genetic algorithms instead of brute force"
 url="http://lcamtuf.coredump.cx/afl/"
-arch="all !armhf"
+arch="all !x86"
 license="Apache-2.0"
+options="!check"
```
Comment (TBK) [inline community/afl/APKBUILD]: Could I get you to add a comment on why check is disabled?

**Comment by maxice8** [comment]
ping

**Comment by J0WI** [comment]
Rebased on top of d30c13e2e02fb35f398354e71d19ecb4bead5b18

**Comment by maxice8** [comment]
Merged, can be closed.

**Comment by J0WI** [comment]
ff6da76883a3bf4d869ae812cd9b075085c981cc

**Pull request closed without merging**